engine/src/test/resources/workflows/invalid_process_condition.bpmn

```diff
@@ -1,13 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:zeebe="http://camunda.org/schema/zeebe/1.0" id="Definitions_1" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="1.8.2">
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL"
+  xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Definitions_1"
+  targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="1.8.2">
   <bpmn:process id="process" name="Process" isExecutable="true">
-  <bpmn:startEvent id="start" />
-  <bpmn:sequenceFlow id="flow1" sourceRef="start" targetRef="gateway" />
-  <bpmn:exclusiveGateway id="gateway" />
-  <bpmn:sequenceFlow id="flow2" sourceRef="gateway" targetRef="end" >
-    <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">foo</bpmn:conditionExpression>
-  </bpmn:sequenceFlow>
-  <bpmn:endEvent id="end" />
+    <bpmn:startEvent id="start"/>
+    <bpmn:sequenceFlow id="flow1" sourceRef="start" targetRef="gateway"/>
+    <bpmn:exclusiveGateway id="gateway"/>
+    <bpmn:sequenceFlow id="flow2" sourceRef="gateway" targetRef="end">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=?!</bpmn:conditionExpression>
```

Comment:
What if the condition is `foo`? Is it a valid condition expression?

Reply:
Yes, it is. `foo` could be a boolean variable that is computed before. 
Currently, we need to write 'foo == true` :sweat_smile:

Reply:
So it is always treated as an expression to be evaluated, not just a static value foo.

Reply:
Correct :+1: 
I see no use case for a static value here.